oc2edr.md

```diff
@@ -497,19 +497,35 @@ Arguments provide additional precision to a Command by including information suc
 | 3 | ipv6_net | ArrayOf(IPv6-Net) | 0..1 | The IPv6 address(es) or range(s) the contained device(s) can still communicate with. |
 
 ### 2.1.5 Actuator Specifiers
-An Actuator is the entity that provides the functionality and performs the Action. The Actuator executes the Action on the Target. In the context of this profile, the Actuator is the EDR and the presence of one or more Specifiers further refine which Actuator(s) shall execute the Action.
+An Actuator is the entity that provides the functionality and performs the Action. The Actuator executes the Action on the Target. In the context of this profile, the Actuator is the ER and the presence of one or more Specifiers further refine: 
 
-The Actuator Specifiers defined in this document are referenced under the `edr` namespace.
+1. Which Actuator(s) shall execute the Action 
+2. On which specific entity or list of entities the Actuator(s) shall execute the Action.
 
-**Table 2.1.5-1. EDR Specifiers**
+The Actuator Specifiers defined in this document are referenced under the `er` namespace.
+
+**Table 2.1.5-1. ER Specifier Type Definitions**
+
+**_Type: ER-server_**
+| ID | Name | Type | # | Description |
+| :--- | :--- | :--- | :--- | :--- |
+| 1 | **hostname** | Hostname | 1 | The hostname of a machine with ER Actuator capabilities. MUST be formatted as an internet host name as specified in [[RFC1123]](#rfc1123) |
+| 2 | **tenant_id** | string | 0..1 | description |
+
+**_Type: ER-endpoint_**
+| ID | Name | Type | # | Description |
+| :--- | :--- | :--- | :--- | :--- |
+| 1 | **endpoint_id** | string | 0..1 | Unique identifier for a particular device with an EDR sensor installed. |
```

Comment:
Do you want this to say "EDR sensor" or "ER sensor"?  We've renamed the AP, but I recognize the product space is still EDR. If there isn't clarifying language about that somewhere in section 1 we should consider adding some.